src/styles/sidebar/components/annotation-body.scss

```diff
@@ -38,6 +39,7 @@
   justify-content: flex-end;
 
   .annotation-body__collapse-toggle-button {
+    @include buttons.button--labeled;
```

Comment:
So, as a general example here: `Annotation` can pass `annotation-body__collapse-toggle-button` as a `className` to a `Button` and then do whatever overrides to button styling (based on `buttons.button--labeled`) it wants to here.